source/DoubleCache.SystemWebCaching/HttpCache.cs

```diff
@@ -1,14 +1,21 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
 using System.Threading.Tasks;
 using System.Web.Caching;
 
-namespace DoubleCache.SystemWebHttpCache
+namespace DoubleCache.SystemWebCaching
 {
     public class HttpCache : ICacheAside
     {
+        internal class CacheItemWrapper
```

Comment:
You have this same internal class in a few different caches. Perhaps pull it out into a separate file so you can share it between classes?